examples/new_maps.py

```diff
@@ -0,0 +1,289 @@
+sens=1
+knee=1
+ylf=1
+fsky=0.1
+nell=np.zeros([nfreq,lmax+1])
+_,nell[:,2:],_=Simons_Observatory_V3_SA_noise(sens,knee,ylf,fsky,lmax+1,1)
+nell*=cl2dl[None,:]
+
+N_ells_sky = np.zeros([nfreq, npol, nfreq, npol, nells])
+for i,n in enumerate(nu):
+    for j in [0,1]:
+        N_ells_sky[i, j, i, j, :] = nell[i]
+
+# Do simulation
+npix = hp.nside2npix(nside)
+maps_comp = np.zeros([ncomp, npol+1, npix])
+maps_comp[0,:,:] = hp.synfast([cl_cmb_tt, cl_cmb_ee, cl_cmb_bb, cl_cmb_te, cl_cmb_eb, cl_cmb_tb], nside, new=True)
+maps_comp[1,:,:] = hp.synfast([cl_dust_tt, cl_dust_ee, cl_dust_bb, cl_dust_te, cl_dust_eb, cl_dust_tb], nside, new=True)
+maps_comp[2,:,:] = hp.synfast([cl_sync_tt, cl_sync_ee, cl_sync_bb, cl_sync_te, cl_sync_eb, cl_sync_tb], nside, new=True)
+
+if o.use_pysm:
+    d2 = models("d2", nside) 
+    s1 = models("s1", nside) 
+    c1 = models("c1", nside) 
+
+    map_I_dust, map_Q_dust, map_U_dust = maps_comp[1,:,:]
+    map_I_sync, map_Q_sync, map_U_sync = maps_comp[2,:,:]
+    map_I_cmb,map_Q_cmb,map_U_cmb = maps_comp[0,:,:]
+
+    # Dust
+    d2[0]['A_I'] = map_I_dust
+    d2[0]['A_Q'] = map_Q_dust
+    d2[0]['A_U'] = map_U_dust
+    d2[0]['spectral_index'] = beta_dust
+    d2[0]['temp'] = temp_dust * np.ones(d2[0]['temp'].size) #need array, no const value for temp with PySM
+    # Sync
+    s1[0]['A_I'] = map_I_sync
+    s1[0]['A_Q'] = map_Q_sync
+    s1[0]['A_U'] = map_U_sync
+    s1[0]['spectral_index'] = beta_sync
+    # CMB
+    c1[0]['A_I'] = map_I_cmb
+    c1[0]['model'] = 'pre_computed' #different output maps at different seeds 
+    c1[0]['A_Q'] = map_Q_cmb
+    c1[0]['A_U'] = map_U_cmb
+
+    sky_config = {'dust' : d2, 'synchrotron' : s1, 'cmb' : c1}
+
+    sky = pysm.Sky(sky_config)
+
+    beams=Simons_Observatory_V3_SA_beams() 
+    instrument_config = {
+        'nside' : nside,
+        'frequencies' : nu, #Expected in GHz 
+        'use_smoothing' : o.use_smoothing,
+        'beams' : beams, #Expected in arcmin 
+        'add_noise' : False,
+        'use_bandpass' : False,
+        'channel_names' : ['LF1', 'LF2', 'MF1', 'MF2', 'UHF1', 'UHF2'],
+        'output_units' : 'uK_RJ',
+        'output_directory' : dirname,
+        'output_prefix' : '/test_deltabpass_',
+    }
+    
+    sky = pysm.Sky(sky_config)
+    instrument = pysm.Instrument(instrument_config)
+    maps_signal, _ = instrument.observe(sky, write_outputs=False)
+    maps_signal = maps_signal[:,1:,:]
+    maps_signal = maps_signal/f_cmb_RJ[:,None,None]
+else:
+    sed_comp = np.array([f_cmb, f_dust, f_sync]).T
+    maps_signal = np.sum(maps_comp[None, :,1:,:]*sed_comp[:,:,None,None], axis=1)
+
+# Save map
+nmaps = nfreq * npol
+hp.write_map(dirname+"/maps_sky_signal.fits", maps_signal.reshape([nmaps,npix]),
+             overwrite=True)
+
+# Mask
+nhits=hp.ud_grade(hp.read_map("norm_nHits_SA_35FOV.fits",  verbose=False),nside_out=nside)
+nhits/=np.amax(nhits) 
+fsky_msk=np.mean(nhits) 
+nhits_binary=np.zeros_like(nhits) 
+nhits_binary[nhits>1E-3]=1 
```

Comment:
Let's make this a bit higher. Maybe something like 0.1. Can you check how much fraction of the sky we're left with?

Reply:
Actually, ignore this comment, please

Reply:
Fixed this. fsky_msk is 0.098